src/components/BottomNavBar/BottomNavBar.js

```diff
@@ -29,12 +29,7 @@ function SideBar() {
   };
 
   // Handles functionality when create goal tab is clicked
-  const createGoal = () => {
```

Comment:
We need this function. It creates a new goal in the database and the saga then fetches it to set it to the create goal page.